examples/templates/ecs-container/main.tf

```diff
@@ -0,0 +1,242 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 4.28"
+    }
+    coder = {
+      source  = "coder/coder"
+      version = "~> 0.4.9"
+    }
+    cloudinit = {
+      source  = "hashicorp/cloudinit"
+      version = "2.2.0"
+    }
+  }
+}
+
+# required for multi-user data config
+provider "cloudinit" {}
+
+# configure AWS provider with creds present on Coder server host
+provider "aws" {
+  region                   = "us-east-1"
+  shared_config_files      = ["/home/coder/.aws/config"]
+  shared_credentials_files = ["/home/coder/.aws/credentials"]
```

Comment:
Would suggest commenting this out. Terraform will do `$HOME/.aws/config` and `$HOME/.aws/credentials` by default and will result in errors if the user is not named `coder`.